src/Bicep.Cli.IntegrationTests/ProgramTests.cs

```diff
@@ -132,7 +129,19 @@ public void BuildSingleFileShouldProduceExpectedTemplate(DataSet dataSet)
             {
                 result.Should().Be(0);
                 output.Should().BeEmpty();
-                error.Should().BeEmpty();
+
+                if (dataSet.Name == "Parameters_LF" || dataSet.Name == "Parameters_CRLF")
```

Comment:
>dataSet [](start = 20, length = 7)

Should we refactor both of the additions in this file into a common method?

Reply:
Good pt. Will extract them into a method.